tutorials/migration_joint_api.md

```diff
@@ -0,0 +1,192 @@
+\page migrationjointapi
```

Comment:
Dis you add this tutorial to an index?

Reply:
ah just added in [87c7a2e](https://github.com/gazebosim/gz-sim/pull/1925/commits/87c7a2e05197c9d7ee67c6b376865375e2261bd9)